src/custom-typings.d.ts

```diff
@@ -43,9 +43,6 @@ import * as _ from 'lodash'
  *
  */
 
-// support NodeJS modules without type definitions
-declare module '*';
```

Comment:
we need this for typescript 2.0 so the experience for installing new modules isn't so crazy.